HACKING/HACKING.txt

```diff
@@ -1,6 +1,5 @@
 Content-Type: text/x-zim-wiki
 Wiki-Format: zim 0.4
-Creation-Date: 2014-07-31T20:56:17+02:00
```

Comment:
On merge will need to put back these creation dates - unintended change due to bug in zim 0.66 I suppose